Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -0,0 +1,41 @@
+Enable Hubble
+==============
+
+Hubble is a fully distributed networking and security observability platform
+for cloud native workloads. It is built on top of Cilium and eBPF to enable
+deep visibility into the communication and behavior of services as well as the
+networking infrastructure in a completely transparent manner.
+
+Deploy Hubble using Helm:
+
+.. parsed-literal::
+
+   helm upgrade cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set global.hubble.enabled=true \\
+      --set global.hubble.listenAddress=":4244" \\
+      --set global.hubble.metricsServer=":54321" \\
```

Comment:
Are these necessary? Any reason we avoid just using the defaults here?

Reply:
-  `--set global.hubble.listenAddress=":4244"` this one is necessary for hubble-relay because by default hubble only listens on a unix domain socket.
- `--set global.hubble.metricsServer=":54321"` i guess this is also necessary, hubble metrics is disabled by default: https://github.com/cilium/cilium/blob/8ba1f84d2ffe5d59b1ca0a94c8ab804a9d43feac/install/kubernetes/cilium/values.yaml#L463

Reply:
removed `global.hubble.metricsServer`. i think we need to live with `global.hubble.listenAddress` for now.